spec/requests/analytics/sales_spec.rb

```
@@ -159,6 +159,84 @@
+      expect(page.current_url).to include("from=")
+      expect(page.current_url).to include("to=")
+      
+      # Verify the date picker text changed to show the last quarter range
+      last_quarter_date_picker_text = find('[aria-label="Date range selector"]').text
+      expect(last_quarter_date_picker_text).not_to eq(initial_date_picker_text)
+      expect(last_quarter_date_picker_text).not_to eq(quarter_date_picker_text)
+    end
+
+    it "handles quarterly date ranges and verifies quarterly options are present" do
+      visit sales_dashboard_path
+      
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)